postgresql/tox.ini

```diff
@@ -14,6 +14,14 @@ deps =
     pytest-catchlog
     pytest-flakes
     pytest-pep8
+    -e
+    {toxinidir}/../common
+    -e
+    {toxinidir}/../server
```

Comment:
putting "-e" and the location in different lines is intentional?   I thought it needs to be on the same line.

Reply:
Not a bug - a documented limitation: "Each line specifies a file, a URL or a package name." [*](http://tox.readthedocs.io/en/latest/config.html#confval-deps=MULTI-LINE-LIST) (so, no options) - there is no way to add options as each line is taken as is and not split before passed to pip. If you need options you either need to use a requirements file or stick you pip invocation directly into commands.

But you worked around this limitation beautifully :)

Reply:
Options could be supported by doing (pseudocode):
```python
if option.startswith('-'):
    parts = option.split(None, 1)
    args.extend(parts)
```
or am I missing something?

Reply:
I agree that it seems fixable with a bit of extra effort. Question is, if someone wants it strong enough to add it. I never missed it and was not even aware of that limitation until I saw this :)

Reply:
True

Reply:
btw sorry for necromancing this - I just realized that the last comment here was months ago :D